tests/utils/delay.test.ts

```diff
@@ -0,0 +1,40 @@
+import { expect } from 'chai';
+import { delay, isPromiseLike } from '../../src';
+
+describe('delay', () => {
+
+   it('delays before returning a value', function(done) {
```

Comment:
Not a big deal, but just wondering why use an arrow function in the `describe` call above but `function` in this `it` and the one below. Probably just force of habit?

Reply:
haha ... no, I started with arrow functions and then had a weird error with the async tests not working and started barking up the wrong tree by changing them to regular functions ...

reverted back to arrow functions:

```
jrthomer@work ~/code/silvermine/toolbox
[ add-promise-delay ] $ git diff 9f7ff9a..8f5297b
diff --git a/tests/utils/delay.test.ts b/tests/utils/delay.test.ts
index 4443440..81fd63e 100644
--- a/tests/utils/delay.test.ts
+++ b/tests/utils/delay.test.ts
@@ -3,7 +3,7 @@ import { delay, isPromiseLike } from '../../src';

 describe('delay', () => {

-   it('delays before returning a value', function(done) {
+   it('delays before returning a value', (done) => {
       let after: string | null = null,
           promise = delay(10, 'this-is-my-test-value').then((v) => { after = v; });

@@ -20,7 +20,7 @@ describe('delay', () => {
       }, 15);
    });

-   it('returns undefined when no value supplied', function(done) {
+   it('returns undefined when no value supplied', (done) => {
       let after: string | undefined = 'initial-value',
           promise = delay(10).then((v) => { after = v; });
```